translations/code-of-conduct.hindi.md

```diff
@@ -0,0 +1,45 @@
+# हसुरा ग्रेफ्लिक इंजन समुदाय आचार संहिता
+
+## हमारी प्रतिज्ञा
+
+एक खुले और स्वागतयोग्य वातावरण को बढ़ावा देने के हित में, हम योगदानकर्ताओं और अनुरक्षकों के रूप में अपनी परियोजना और हमारे समुदाय में सभी के लिए एक उत्पीड़न मुक्त अनुभव बनाने की प्रतिज्ञा करते हैं, भले ही उम्र, शरीर का आकार, विकलांगता, जातीयता, सेक्स विशेषताओं, लिंग पहचान और अभिव्यक्ति, अनुभव का स्तर, शिक्षा, सामाजिक-आर्थिक स्थिति, राष्ट्रीयता, व्यक्तिगत उपस्थिति, जाति, धर्म या यौन पहचान या अभिविन्यास
+
+## हमारे मानक
+
+सकारात्मक वातावरण बनाने में योगदान देने वाले व्यवहार के उदाहरणों में शामिल हैं:
+
+* स्वागतयोग्य, समावेशी और लैंगिक-तटस्थ भाषा का उपयोग करना (उदाहरण: "हे लोग" के बजाय, आप "हे लोग" या "हे हेम" का उपयोग कर सकते हैं) 
+* विभिन्न दृष्टिकोणों और अनुभवों के प्रति सम्मानजनक होना
+* रचनात्मक आलोचना स्वीकार करते हुए
+* समुदाय के लिए सबसे अच्छा है पर ध्यान केंद्रित करना
+* अन्य समुदाय के सदस्यों के प्रति सहानुभूति दिखाना
+
+प्रतिभागियों द्वारा अस्वीकार्य व्यवहार के उदाहरणों में शामिल हैं:
+
+* यौन भाषा या कल्पना और अवांछित यौन ध्यान या अग्रिमों का उपयोग
+* ट्रोलिंग, अपमानजनक / अपमानजनक टिप्पणी, और व्यक्तिगत या राजनीतिक हमले
+* सार्वजनिक या निजी उत्पीड़न
+* दूसरों की निजी जानकारी, जैसे कि एक भौतिक या इलेक्ट्रॉनिक पता, बिना स्पष्ट अनुमति के प्रकाशित करना
+* अन्य आचरण जो एक पेशेवर सेटिंग में उचित रूप से अनुचित माना जा सकता है
+
+## हमारी जिम्मेदारियाँ
+
+परियोजना अनुचर स्वीकार्य व्यवहार के मानकों को स्पष्ट करने के लिए जिम्मेदार हैं और अस्वीकार्य व्यवहार के किसी भी उदाहरण के जवाब में उचित और उचित सुधारात्मक कार्रवाई करने की उम्मीद है।
+
+प्रोजेक्ट मेंटेनर के पास टिप्पणियों को हटाने, संपादित करने, या अस्वीकार करने, संपादित करने, कोड करने, विकी संपादन, मुद्दे और अन्य योगदान करने का अधिकार और जिम्मेदारी है, जो इस आचार संहिता से जुड़े नहीं हैं, या अन्य व्यवहारों में अस्थायी या स्थायी रूप से किसी भी योगदानकर्ता पर प्रतिबंध लगाने के लिए वे अनुचित, धमकी, अपमानजनक या हानिकारक हैं।
+
+## क्षेत्र
+
+यह आचार संहिता परियोजना के भीतर और सार्वजनिक स्थानों पर लागू होती है जब कोई व्यक्ति परियोजना या उसके समुदाय का प्रतिनिधित्व करता है। एक परियोजना या समुदाय का प्रतिनिधित्व करने के उदाहरणों में एक आधिकारिक परियोजना ई-मेल पते का उपयोग करना, एक आधिकारिक सोशल मीडिया खाते के माध्यम से पोस्ट करना, या एक ऑनलाइन या ऑफ़लाइन कार्यक्रम में एक नियुक्त प्रतिनिधि के रूप में कार्य करना शामिल है। एक परियोजना का प्रतिनिधित्व आगे परियोजना के रखवालों द्वारा परिभाषित और स्पष्ट किया जा सकता है।
+
+## प्रवर्तन
+
+समुदाय की वेबसाइट community@hasura.io पर संपर्क करके अपमानजनक, परेशान करने या अन्यथा अस्वीकार्य व्यवहार की सूचना दी जा सकती है। सभी शिकायतों की समीक्षा और जांच की जाएगी और परिणामस्वरूप एक प्रतिक्रिया होगी जो परिस्थितियों के लिए आवश्यक और उचित समझी जाएगी। प्रोजेक्ट टीम एक घटना के रिपोर्टर के संबंध में गोपनीयता बनाए रखने के लिए बाध्य है। विशिष्ट प्रवर्तन नीतियों के और विवरण अलग से पोस्ट किए जा सकते हैं।
```

Comment:
Please reframe this paragraph, the sentences are a little out of sync and might need a redo.